server/templates/details.html.tmpl

```diff
@@ -158,7 +158,7 @@
         <ul class="list-disc list-outside pl-6 py-2">
           {{range .Values}}<li class="font-mono text-sm-mono">{{.}}</li>{{end}}
         </ul>
-        {{if eq $s "skipped"}}do not {{end}}{{.ConditionPhrase}}
+        {{if eq $s "skipped"}}{{.SkipPhrase}} {{end}}{{.ConditionPhrase}}
```

Comment:
One issue with using `SkipPhrase` like this is that we'll need to either set a value in _all_ of the existing predicates or provide a way to use a default value when it isn't set.